.github/workflows/ci.yml

```diff
@@ -7,11 +7,6 @@ on:
     types: [ published ]
   pull_request:
     branches: [ main ]
-    paths-ignore:
```

Comment:
I have tried to explain the changes [here](https://github.com/sovity/edc-extensions/pull/926#issuecomment-2124668585), otherwise steps of the PR pipeline are not executed at all, but are at the same time marked as `Required` so that the PR can be merged without using Admin-Powers.

Reply:
as discussed in our teams chat, we could build the CI action like this, so it still runs but if it sees only docs-changes, it goes immediately green and passed like in the following template, with the remaining steps being executed for all other changes (all need this if)

```
      - name: Checkout code
        uses: actions/checkout@v2
 
      - name: Check for docs-only changes
        id: docs-check
        run: |
          if git diff --name-only ${{ github.event.before }} ${{ github.sha }} | grep -E '^(docs/.*|.*\.md|.*\.rst)$'; then
            echo "docs=true" >> $GITHUB_ENV
          else
            echo "docs=false" >> $GITHUB_ENV
          fi
 
      - name: Success if only docs changed
        if: env.docs == 'true'
        run: echo "Only documentation files changed. Marking as successful."
 
      - name: Run tests if other files changed
        if: env.docs == 'false'
        run: echo "Non-documentation files changed. Running tests."
```

Reply:
I'm going to leave it as it is inital here, so that the CI is also executed for documentation changes, as the passing of these jobs is necessary for the the pr-pipeline as a whole to pass. The above attempt does not work without further ado, it results in a wild combination of `fetch-depth: 0` and EDC CE errors with already existing JARs in the Build-Gradle job. In principle, of course, it generally makes sense to speed up what takes a long time, here in this CI-workflow this is the EDC CE Build-Gradle step, which runs for about 9 minutes, perhaps something can be improved on the code side of the EDC CE, whereby all other steps take around 0-20 seconds. https://github.com/sovity/edc-extensions/actions/runs/9267246759/job/25493261518?pr=927